### ISSUE TITLE
Tag 21.3.0-full image won't start on Docker Desktop for Windows

### ISSUE THREAD

**Issue by fabio-grassi-gbs** [gvenzl/oci-oracle-xe#118]
While tag slim and tag regular work well, the tag full won't start under Docker Desktop for Windows 10 version 4.9.1. Containers exits after a few seconds with exit code 187. Last line logged says "ORA-00443: background process "GEN1" did not start".

The full log is the following:

************ LOG START ************
```
CONTAINER: starting up...
CONTAINER: database already initialized.
CONTAINER: starting up Oracle Database...

LSNRCTL for Linux: Version 21.0.0.0.0 - Production on 02-JUL-2022 17:35:13

Copyright (c) 1991, 2021, Oracle.  All rights reserved.

Starting /opt/oracle/product/21c/dbhomeXE/bin/tnslsnr: please wait...

TNSLSNR for Linux: Version 21.0.0.0.0 - Production
System parameter file is /opt/oracle/homes/OraDBHome21cXE/network/admin/listener.ora
Log messages written to /opt/oracle/diag/tnslsnr/1be72b33af2f/listener/alert/log.xml
Listening on: (DESCRIPTION=(ADDRESS=(PROTOCOL=ipc)(KEY=EXTPROC_FOR_XE)))
Listening on: (DESCRIPTION=(ADDRESS=(PROTOCOL=tcp)(HOST=0.0.0.0)(PORT=1521)))

Connecting to (DESCRIPTION=(ADDRESS=(PROTOCOL=IPC)(KEY=EXTPROC_FOR_XE)))
STATUS of the LISTENER
------------------------
Alias                     LISTENER
Version                   TNSLSNR for Linux: Version 21.0.0.0.0 - Production
Start Date                02-JUL-2022 17:35:13
Uptime                    0 days 0 hr. 0 min. 0 sec
Trace Level               off
Security                  ON: Local OS Authentication
SNMP                      OFF
Default Service           XE
Listener Parameter File   /opt/oracle/homes/OraDBHome21cXE/network/admin/listener.ora
Listener Log File         /opt/oracle/diag/tnslsnr/1be72b33af2f/listener/alert/log.xml
Listening Endpoints Summary...
  (DESCRIPTION=(ADDRESS=(PROTOCOL=ipc)(KEY=EXTPROC_FOR_XE)))
  (DESCRIPTION=(ADDRESS=(PROTOCOL=tcp)(HOST=0.0.0.0)(PORT=1521)))
The listener supports no services
The command completed successfully
ORA-00443: background process "GEN1" did not start
```
************ LOG END ************

**Comment by gvenzl** [comment]
Hi @fabio-grassi-gbs,

For some reason, the database didn't manage to startup one of its background processes. We will have to find out why that was the case. I googled a bit around and found some old threads hinting that this could be related to not having enough resources available. But these entries on the web are very old so it's just an indication, nothing more, see [here](https://community.oracle.com/tech/developers/discussion/2606965/ora-00443-background-process-pmon-did-not-start/p2).

We will need more information to find out what's really going on. Could you please provide me with the (content of) the alert log file. It's inside the container under `/opt/oracle/diag/rdbms/xe/XE/trace/alert_XE.log`. To get to it, you will probably have to overwrite the container entrypoint, manually run the entrypoint script and then get the contents of the file.

This should do the trick:

1) `$ docker run -ti --name test --entrypoint=bash gvenzl/oracle-xe:full`
1) `$ export ORACLE_RANDOM_PASSWORD=yes`
1) `$ ./container-entrypoint.sh`
1) `$ cat /opt/oracle/diag/rdbms/xe/XE/trace/alert_XE.log`
1) `$ exit`
1) `$ docker rm test`

**Comment by fabio-grassi-gbs** [comment]
SOLVED! I followed your instructions and I discovered that now the container started properly. I compared what I had been doing with your instructions and I realized that the only real difference was that I was using the -v option in order to have a persistent database. I then tried to run the same image without volume mapping and everything worked. So I purged unused volumes and run again the full image with volume mapping and this time it worked like a charm. It is now clear to me that I just had hit the volume size limit and that there is no problem with the full image. Sorry for bothering you and thank you for your prompt response and very much appreciated help.
